tempesta_fw/t/functional/wrk/connect.lua

```diff
@@ -0,0 +1,2 @@
+wrk.method = "CONNECT"
```

Comment:
`Connect` is very specific method: after 200 response connection between client and server works as a pipe, not as HTTP connection. We don't support the method. What the test checks?

Reply:
the test checks not failing of tempesta under multiple connect requests